nixos/modules/services/x11/window-managers/exwm.nix

```
@@ -38,6 +38,10 @@ in
           file.
         '';
       };
+      package = mkPackageOption pkgs "Emacs" {
+        default = "emacs";
+        example = "pkgs.emacs-gtk";
```

```suggestion
        example = [ "emacs-gtk" ];
```

This is a bit better because we avoid duplicate `pkgs` (`pkgsText` actually).